tests/ui/lto/dwarf-mixed-versions-lto.rs

```diff
@@ -4,7 +4,7 @@
 
 //@ ignore-msvc Platform must use DWARF
 //@ aux-build:dwarf-mixed-versions-lto-aux.rs
-//@ compile-flags: -C lto -g -Zdwarf-version=5
+//@ compile-flags: -C lto -g -Cdwarf-version=5
```

Comment:
Discussion (re. call-for-testing period): should we still explicitly have a call-for-testing period, to invite users to *explicitly* try to break this flag somehow? It seems not too high risk, but still.

> No call-for-testing has been conducted but Rust for Linux has been using this flag without issue.

RfL is an important user group, but they may have their specific ways of exercising this flag. Would we benefit from having more diverse user groups try to break this?

Reply:
No objection to a call-for-testing period but I think we've gotten most of the interesting user testing done by RfL here as the Linux kernel developer community cares quite a lot about things like the various new compact forms in DWARF 5.

Reply:
That's good enough for me, I'll leave this unresolved for a brief moment, in case someone else has concerns. But feel free to resolve this (non-blocking).